client-app/shared/catalog/components/product/properties.vue

```diff
@@ -18,15 +18,29 @@
     >
       <Vendor :vendor="product.vendor" withRating />
     </ProductProperty>
+
+    <a
+      v-if="product.properties && product.properties.length > MAX_DISPLAY_ITEMS"
+      class="flex gap-x-1 mb-4 items-center text-14 text-[color:var(--color-link)] hover:text-[color:var(--color-link-hover)] hover:cursor-pointer underline decoration-dashed"
+      @click="showAll = !showAll"
+    >
+      <VcIcon class="text-[color:var(--color-primary)]" :name="[showAll ? 'chevron-up' : 'chevron-down']" size="xs" />
+      {{
+        showAll == true ? $t("shared.catalog.product_details.see_less") : $t("shared.catalog.product_details.see_more")
+      }}
+    </a>
   </ProductTitledBlock>
 </template>
 
 <script setup lang="ts">
-import { computed, PropType } from "vue";
+import { computed, PropType, ref } from "vue";
 import _ from "lodash";
 import { Product } from "@/xapi/types";
 import { prepareProperties, ProductProperty, ProductTitledBlock, Vendor } from "@/shared/catalog";
 
+const MAX_DISPLAY_ITEMS = 8;
+const showAll = ref(false);
+
 const props = defineProps({
```

Comment:
mb change 'const props = defineProps(...' to ower interface solution? [like here](https://github.com/VirtoCommerce/vc-theme-b2b-vue/pull/432/files#diff-bfd252f92c636f553a414b5b0033155be478d69dae522045afad8719879f23e5:~:text=%7D)%3B-,interface%20Props%20%7B,-cart%3A%20CartType)